tools/openblas_support.py

```diff
@@ -231,6 +231,26 @@ def get_members():
             yield member
 
     tarfileobj.extractall(target_path, members=get_members())
+    reformat_pkg_file(target_path=target_path)
+
+
+def reformat_pkg_file(target_path):
```

Comment:
minor: if you're pushing another comment, perhaps add a comment that this is simple typo that is already fixed upstream in `openblas#4594`